### PR TITLE
Add TMAP8 getting started documentation and clean-up homepage

### LINKED ISSUE #19: TMAP8 needs getting started instructions
TMAP8 does not have getting started instructions for users who are handed the URL to the code documentation site. This should be written and a clean-up of the homepage should be performed so that users can get started more easily.

### PR DESCRIPTION
**Pull request by cticenhour**
@singhgp4321 @lindsayad here are the new getting started documentation pages I was thinking of when meeting with the INL Fusion Safety team. I also cleaned up the homepage and moved Alex's Scaling notes to a "User Notes" page - perhaps we can keep this page updated with good usage guidelines in the future? 

Closes #19

### CODE REVIEW

File: doc/content/getting_started/installation.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+!alert-end!
+
+## Step Three: Build and Test TMAP8
+
+!style halign=left
+To compile TMAP8, first make sure that the conda MOOSE environment is activated
+(*and be sure to do this any time that a new Terminal window is opened*):
+
+```bash
+conda activate moose
```
Comment (lindsayad) [inline doc/content/getting_started/installation.md:58]:
```suggestion
mamba activate moose
```

**Review by lindsayad — CHANGES_REQUESTED** [review]
looks great overall!

File: doc/content/getting_started/installation.md (hunk 1 above)
Comment (cticenhour) [reply doc/content/getting_started/installation.md:58]: Oops good call! 😄

**Review by lindsayad — APPROVED** [review]

**Comment by singhgp4321** [comment]
Thanks for adding this @cticenhour !

**Pull request merged**